### ISSUE TITLE
getting positions/velocities on the whole system vs. one particle

### ISSUE THREAD

**Issue by rkurchin** [JuliaMolSim/AtomsBase.jl#14]
Just going to file an issue for discussing this since @Gregstrq and I were just going back and forth about it on Slack.

Currently, the setup is (I'm going to write it out just for positions, but it follows exactly analogously for velocities) that there's just one function `position`, which behaves this way:

* `position(::AbstractParticle)` --> position of that particle (this is obvious)
* `position(::AbstractSystem)` --> positions over every particle in the system

Grigory (assuming I was understanding correctly) was suggesting that we could instead do something like
* `position(::AbstractParticle)` --> position of that particle (this is obvious)
* `position(::AbstractSystem, index)` --> position of particle at an index
* `positions(::AbstractSystem)` --> new function for getting _all_ the positions

Personally, I don't feel it's crucial to add a new function name as I think the behavior as it exists now is fairly intuitive, though I would be fine with adding the indexed version.

Anyway, discuss.

**Comment by Gregstrq** [comment]
And for the function that returns all the positions in the system, the name `positions` transmits the intent (that we want multiple positions) much better than `position`.

**Comment by mfherbst** [comment]
I don't care much and I don't mind having the two-argument version of `position` around. My only worry (and the reason for choosing only singular function name) was to avoid confusion when using the interface.

I have experienced more than enough in other packages that the use of `s` to indicate "multiple things returned" or "list returned" was inconsistent such that I had to constantly ask myself: Does this have an `s` or not in the name? So having just one version without the `s` solves this problem once and for all. 

In any case, what is crucial to me is that we carry this interface consistently over to the other related quantities @rkurchin  mentions (`velocities` the no infamous non-static (partial) `charges` etc.)
